core/src/main/java/org/arquillian/smart/testing/Configuration.java

```diff
@@ -2,27 +2,34 @@
 
 public class Configuration {
 
+    public static final String DEFAULT_MODE = "ordering";
+    public static final String DEFAULT_STRATEGIES = "";
+
     public static final String SMART_TESTING = "smart.testing";
     public static final String SMART_TESTING_MODE = "smart.testing.mode";
     public static final String SMART_TESTING_PLUGIN = "smart.testing.plugin";
```

Comment:
well I will not do it CSV because by default they are both, so user will only use this property to  select one.